### DISCUSSION TITLE
Overture Maps Foundation Daylight Distribution

### DISCUSSION THREAD

**Discussion by alext951** [organicmaps/organicmaps#5910] [Ideas]
[Daylight distribution](https://daylightmap.org/) is a free downstream distribution of OSM with cleaned up road intersections, simplified geometry for routing, shape consistency checks, and checks against vandalism. They release a planetary file in OSM's PBF format once a month. Does it make sense to consider using this for navigation with Organic Maps? I see the benefits, and the format appears to be compatible, the distribution is free - so I'd be curious to hear both how I could possibly give it a try (is it using the map generator?) and also why it may not be a good idea. Thanks

**Comment by biodranik** [comment]
Please check existing discussions and issues about this topic. We also discussed it in Telegram/Matrix.

**Reply by alext951** [reply]
Apologies, I couldn't find it. Will take another pass, thanks for responding.

**Comment by map-per** [comment]
Yes, in theory you can use the generator_tool to to create map data from .osm.pbf files. But the planet file that Daylight provides is to big for a nomal PCs. I'd start with smaller regions from Geofabrik (see https://github.com/organicmaps/organicmaps/blob/ef67cd04f1d954e6ba19bf45ba671c3f793c9f06/tools/python/maps_generator/README.md).
